diff --git a/IntelPresentMon/PresentMonAPIWrapper/EtlLogger.cpp b/IntelPresentMon/PresentMonAPIWrapper/EtlLogger.cpp
index 0bfda88..e05f013 100644
--- a/IntelPresentMon/PresentMonAPIWrapper/EtlLogger.cpp
+++ b/IntelPresentMon/PresentMonAPIWrapper/EtlLogger.cpp
@@ -1,6 +1,5 @@
 #include "EtlLogger.h"
-#include "../CommonUtilities/log/Log.h"
-#include "../PresentMonAPIWrapperCommon/Exception.h"
+#include <IntelPresentMon/PresentMonAPIWrapperCommon/Exception.h>
 
 namespace pmapi
 {
